app/presenters/user_presenter.rb

```diff
@@ -24,14 +24,14 @@ def services
   end
 
   def configured_services
-    user.services.map{|service| service.provider }
+    user.services.map(&:provider)
   end
 
   def aspects
     @aspects ||= begin
                    aspects = AspectPresenter.as_collection(user.aspects)
-                   no_aspects = self.aspects_ids.empty?
-                   aspects.each{ |a| a[:selected] = no_aspects || self.aspects_ids.include?(a[:id].to_s) }
+                   no_aspects = aspects_ids.empty?
+                   aspects.each {|a| a[:selected] = no_aspects || aspects_ids.include?(a[:id].to_s) }
```

Comment:
This "each" is pretty odd. Or maybe it's because I don't know the domain language. What is an "aspect" when it comes to an user?